frappe/email/doctype/email_account/email_account.py

```diff
@@ -288,6 +288,12 @@ def set_thread(self, communication, email):
 			communication.reference_doctype = parent.doctype
 			communication.reference_name = parent.name
 
+		# check if message is notification and disable notifications for this message
+		references = email.mail.get("References")
+		if references:
+			if "notification" in references:
+				communication.unread_notification_sent = 1
+
```

Comment:
notification is added to header here
https://github.com/frappe/frappe/pull/2458/files#diff-a95ae98f6737046bfb40cb1fd19689f8R191
so if email is directly recieved by an erpnext server it set's it so the server believes notifcations are sent